core/src/main/scala/com/nvidia/spark/rapids/tool/analysis/AppSparkMetricsAggTrait.scala

```diff
@@ -59,7 +61,8 @@ trait AppSparkMetricsAggTrait extends AppIndexMapperTrait {
         agg1.sqlAggs ++ agg2.sqlAggs,
         agg1.ioAggs ++ agg2.ioAggs,
         agg1.sqlDurAggs ++ agg2.sqlDurAggs,
-        agg1.maxTaskInputSizes ++ agg2.maxTaskInputSizes)
+        agg1.maxTaskInputSizes ++ agg2.maxTaskInputSizes,
```

Comment:
Q: Why do we not provide the `sqlAnalyzer` when calling `getAggRawMetrics()` here?

Reply:
We only need to provide `sqlAnalyzer` when Qualification calls `getAggRawMetrics`, because `QualificationAppInfo`  does not have `AppSQLPlanAnalyzer`.
`getAggregateRawMetrics` is only called by Profiler, so `getAggRawMetrics` in `getAggregateRawMetrics` is only called by Profiler as well, so we do not need to provide `sqlAnalyzer` here.

Reply:
Thanks @cindyyuanjiang